matrix/helper.go

```diff
@@ -37,17 +37,17 @@ func NewClientSet() (*kubernetes.Clientset, *rest.Config) {
 }
 
 // ValidateOrFail validates connectivity
-func ValidateOrFail(k8s *KubeManager, model *Model, testCase *TestCase, ignoreLoopback bool) int {
+func ValidateOrFail(k8s *KubeManager, model *Model, testCase *TestCase, ignoreLoopback, reachTargetPod bool) int {
```

Comment:
I would stop to add new params (after this one) in the method signature, it's being used all over and it's very annoying to change all the files.

Reply:
We need to refactor when we need to add more cases in validateOrFail